### PR TITLE
Continue with making the schedule ins CSS

### PR DESCRIPTION
**Pull request by sweetstyle11**

### CODE REVIEW

**Comment by sweetstyle11** [comment]
I'm trying to build the day picker dropdown feature. Right now I'm just trying to get the dropdown to appear when clicked, then be able to click on a day, and what ever day that was clicked, I want to be able to hide add other days.  I'm adding event listener's using a loop, and I'm not 100% if this is the right approach. The Javascript looks like this:

var day_picker_box = document.getElementById("day_picker_box");
var element_to_show = document.getElementsByClassName("day_picker__text");

day_picker_box.addEventListener("click", function(){
  for (var i = element_to_show.length - 1; i >= 0; i--) {
    element_to_show[i].classList.remove("day_picker__text--hide")
    element_to_show[i].addEventListener("click", function(){
      alert(event.target.innerHTML)
      if (element_to_show[i] !== event.target){
        element_to_show[i].classList.add("day_picker__text--hide")
      }

    })
  }
});

File: app/assets/javascripts/schedule_dropdown.js [hunk 1]
```diff
@@ -2,8 +2,14 @@ var day_picker_box = document.getElementById("day_picker_box");
 var element_to_show = document.getElementsByClassName("day_picker__text");
 
 day_picker_box.addEventListener("click", function(){
-  alert("You Clicked Me!");
   for (var i = element_to_show.length - 1; i >= 0; i--) {
     element_to_show[i].classList.remove("day_picker__text--hide")
+    element_to_show[i].addEventListener("click", function(){
+      alert(event.target.innerHTML)
```
Comment (halfghaninne) [inline app/assets/javascripts/schedule_dropdown.js]: Here you are referencing an `event` that hasn't been passed to the EventListener function.

**Comment by halfghaninne** [comment]
See if the above helps, and if not, I can come down and take a more in-depth look.

**Comment by sweetstyle11** [comment]
That helped a little. I'm still hung up on how my loop is functioning. Do you mind taking a look at it in person?

**Comment by halfghaninne** [comment]
I don't! Sumeet or I can take a look after yoga.

**Pull request closed without merging**